docker/download.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 
 ARCH="${1}"
-TARGET_DIR="/.compiler_cache/$ARCH"
```

Comment:
Nit: Can we add a check on the number of arguments with an error message? Something like:

```
if [[ $# -ne 2 ]] ; then 
   echo "Usage: $0 ...."
   exit -1;
fi
```

Reply:
Addressed in https://github.com/ConsenSys/solc-typed-ast/pull/97/commits/bc60c2c4f19f074ef26fc2875d4e4b404b218c9f